site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -35,7 +36,10 @@ If the `agones-allocator` service is installed as a `LoadBalancer` [using a stat
 
 Replace the default server TLS certificate with a certificate with CN and subjectAltName. There are multiple approaches to generate a certificate. Agones recommends using [cert-manager.io](https://cert-manager.io/) solution for cluster level certificate management.
 
-In order to use cert-manager solution, first, [install cert-manager](https://cert-manager.io/docs/installation/kubernetes/) on the cluster. Then, [configure](https://cert-manager.io/docs/configuration/) an `Issuer`/`ClusterIssuer` resource and last configure a `Certificate` resource to manage allocator-tls `Secret`.
+In order to use cert-manager solution, first, [install cert-manager](https://cert-manager.io/docs/installation/kubernetes/) on the cluster.
```

Comment:
As long as you are updating this paragraph, can you make it into proper sentences?

e.g. "In order to use the cert-manager solution, first [install cert-manager] on the cluster. Then [configure] a ..."

Reply:
Done. Thanks!